src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1017,7 +1017,7 @@
     <!-- Parameter receiving the value. -->
     <Field Name="Parameter" Type="ParameterSymbol" Null="disallow"/>
     <!-- Expression representing the value. -->
-    <Field Name="Value" Type="BoundExpression" Null="disallow"/>
+    <Field Name="Value" Type="BoundExpression" Null="allow"/>
```

Comment:
I'll double check this. I *think* it needs to allow `Value = null`, so when the `Syntax.xml.Generated.vb` is built it allow the initialiser part to be null, thus not throw an exception.
As I said I'll double check and confirm.